xla/service/spmd/dot_handler.cc

```diff
@@ -1571,12 +1590,6 @@ StatusOr<HloInstruction*> EmitWindowedDotGeneral(
       // Odd number iteration.
       auto second_next_l = next_l;
       auto second_next_r = next_r;
-      cp_input = windowed_op_is_lhs ? next_l : next_r;
```

Comment:
@ezhulenev I removed this CP creation logic since it seems to create 2 chained back to back CPs that have exactly the same parameters, this one here doesn't have any consumer other than the root, i think we can directly route the first CP to the root but im not sure if this would impact other backends or not. Can you check internally to see if this is fine? thanks